app/models/organization/CreditTransactionService.scala

```diff
@@ -0,0 +1,92 @@
+package models.organization
+
+import com.scalableminds.util.accesscontext.DBAccessContext
+import com.scalableminds.util.objectid.ObjectId
+import com.scalableminds.util.tools.{Fox, FoxImplicits}
+import com.typesafe.scalalogging.LazyLogging
+import play.api.libs.json.{JsObject, Json}
+
+import javax.inject.Inject
+import scala.concurrent.ExecutionContext
+
+class CreditTransactionService @Inject()(creditTransactionDAO: CreditTransactionDAO,
+                                         organizationService: OrganizationService)(implicit val ec: ExecutionContext)
+    extends FoxImplicits
+    with LazyLogging {
+
+  def hasEnoughCredits(organizationId: String, creditsToSpent: BigDecimal)(
+      implicit ctx: DBAccessContext): Fox[Boolean] =
+    creditTransactionDAO.getCreditBalance(organizationId).map(balance => balance >= creditsToSpent)
+
+  def getCreditBalance(organizationId: String)(implicit ctx: DBAccessContext): Fox[BigDecimal] =
+    creditTransactionDAO.getCreditBalance(organizationId)
```

Comment:
I’m not a big fan of these single-line redirect functions. I’d rather inject the dao where getCreditBalance is needed.